gpflow/kernels/mo_kernels.py

```diff
@@ -100,7 +113,11 @@ class SeparateIndependent(MultioutputKernel, Combination):
     - Independent: Latents are uncorrelated a priori.
     """
     def __init__(self, kernels, name=None):
```

Comment:
Should this support `active_dims`? :thinking:

Reply:
if you think so, you should open a PR!